### ISSUE TITLE
License question

### ISSUE THREAD

**Issue by mgd020** [jdrouet/mrml#369]
Hi,

Thanks for all the hard work on this project!

I maintain a [Python package](https://github.com/mgd020/mjml-python) that uses this library as implementation, and I'd like to set a license on the package.

The repo is MIT as it only includes code by me and contributors, but I am not sure about the built binaries (derived from your source, extended with my own code).

Can you explain what (if any) conditions are required by your license, in layman's terms, relevant to my situation?

Thanks!

**Comment by jdrouet** [comment]
👋 sorry for the late reply: https://github.com/jdrouet/mrml/pull/388
